zbus/src/message.rs

```diff
@@ -196,6 +196,30 @@ impl Message {
         b.build(body)
     }
 
+    /// Create a message from bytes.
+    ///
+    /// The `fds` parameter is only available on unix. It specifies the file descriptors that
+    /// accompany the message. On the wire, values of the UNIX_FD types store the index of the
+    /// corresponding file descriptor in this vector. Passing an empty vector on a message that
+    /// has UNIX_FD will result in an error.
+    ///
+    /// **Note:** Since the constructed message is not construct by zbus, the receive sequence,
+    /// which can be acquired from [`recv_position`], is not applicable and hence set to `0`.
+    ///
+    /// # Safety
+    ///
+    /// This method is unsafe as bytes may have an invalid encoding.
+    ///
+    /// [`recv_position`]: #method.recv_position
```

Comment:
You don't need to hand-write the link. rustdoc does that for you. See my other comment.

Reply:
Oh, good to know, thanks!